tests/fixtures/body-parameter/expected/Resource/TestResource.php

```diff
@@ -17,7 +17,7 @@ class TestResource extends Resource
     public function testSimpleBodyParameter($testString, $parameters = [], $fetch = self::FETCH_OBJECT)
     {
         $queryParam = new QueryParam();
-        $url        = '/test-simple';
+        $url        = 'http://localhost/test-simple';
         $url        = $url . ('?' . $queryParam->buildQueryString($parameters));
         $headers    = array_merge(['Host' => 'localhost'], $queryParam->buildHeaders($parameters));
```

Comment:
It will not make it harder but it can hide missing configuration, and not a fan of this :/